install_config/router/default_haproxy_router.adoc

```diff
@@ -1488,55 +1458,58 @@ xref:../../architecture/networking/assembly_available_router_plugins.adoc#haprox
 are, by default, exposed or published in
 link:https://prometheus.io/docs/concepts/data_model/[Prometheus format]
 for consumption by external metrics collection and aggregation systems (e.g. Prometheus, statsd).
-Metrics are also available dirctly from the
+Metrics are also available directly from the
 link:https://cbonte.github.io/haproxy-dconv/1.5/configuration.html#9[HAProxy router] in its own CSV format.
+These metrics include the HAProxy native metrics and some controller metrics.
+
+When you create a router using the following command, {product-title} makes metrics available in Prometheus format on the stats-port, default 1936. 
 
-When you create a router, as below,
-====
 ----
 $ oc adm router --service-account=router
 ----
-====
-metrics are automatically available in Prometheus format on the stats-port, default 1936. To suppress metrics collection,
-====
+
+* To get the Prometheus statistics for the exposed metrics:
++
+Launch the stats window using the following URL in a browser:
++
 ----
-$ oc adm router --service-account=router --stats-port=0
+http://admin:<Password>@<router_IP>:1936/metrics
 ----
-====
-
-To switch to the HAProxy CSV format metrics, edit the xref:../../architecture/networking/routes.adoc#env-variables[environment variables]
-for the router dc and delete the following lines:
-
-====
++
+To get the router IP, admin name, and password:
++
 ----
-          - name: ROUTER_LISTEN_ADDR
-            value: 0.0.0.0:1936
-          - name: ROUTER_METRICS_TYPE
-            value: haproxy
+oc describe pod <router_pod>
 ----
-====
-Where 1936 is the STATS_PORT value.
-
-You can extract the raw statistics in Prometheus format by using the following.
 
-Information needed to access the metrics is found the router service annotations:
+* To suppress metrics collection,
++
+----
+$ oc adm router --service-account=router --stats-port=0
+----
 
-====
+* To extract the raw statistics in Prometheus format use the following.
```

Comment:
s/format/format,
maybe s/use the following./run the following command and modify the output: